apps/meteor/packages/rocketchat-i18n/i18n/en.i18n.json

```
@@ -4699,6 +4699,13 @@
   "Video_message": "Video message",
   "Videocall_declined": "Video Call Declined.",
   "Video_and_Audio_Call": "Video and Audio Call",
+  "video_conference_ended": "__username__ started __conference__ video conference.",
```

will be there an ended event for video conference?
```suggestion
  "video_conference_ended": "__username__ ended __conference__ video conference.",
```

---

Actually the message is the same in both cases, it's only the uikit blocks that are going to be different, so there is no need for an extra message type here, I'll remove this one.